README_webassembly.md

```diff
@@ -104,108 +90,45 @@ the Halide library itself.
 ## Enabling wasm JIT
 
 If you want to run `test_correctness` and other interesting parts of the Halide
-test suite (and you almost certainly will), you'll need to install libV8 and
-ensure that LLVM is built with wasm-ld:
+test suite (and you almost certainly will), you'll need to ensure that LLVM is built with wasm-ld:
 
 - Ensure that you have lld in LVM_ENABLE_PROJECTS:
 
 ```
 cmake -DLLVM_ENABLE_PROJECTS="clang;lld" ...
 ```
 
-- Install libv8 and the d8 shell tool (instructions omitted), or build from
-  source if you prefer (instructions omitted). We require version v7.5+.
-
-Note that using shared-library builds of V8 may be problematic on some platforms
-(e.g. OSX) due to libc++ conflict issues; using a static-library version may be
-simpler for those. Also note that (as of this writing), libV8 v7.5+ may not be
-available in prebuilt form for your platform.
-
-- Set `V8_INCLUDE_PATH` and `V8_LIB_PATH` to point to the paths for V8 include
-  files and library, respectively.
-
-- Set `WITH_V8=1`
-
-- To run the JIT tests, set `HL_JIT_TARGET=wasm-32-wasmrt` (or
-  `HL_JIT_TARGET=wasm-32-wasmrt-wasm_simd128`) and run normally. The test suites
-  which we have vetted to work include correctness, performance, error, and
-  warning. (Some of the others could likely be made to work with modest effort.)
+- To run the JIT tests, set `HL_JIT_TARGET=wasm-32-wasmrt` (possibly adding `wasm_simd128`, `wasm_signext`, and/or `wasm_sat_float_to_int`) and run CMake/CTest normally. Note that wasm testing is only support under CMake (not via Make).
```

Comment:
line wrapping in this file seems necessary. I think @alexreinking used a markdown auto-formatter at some point that might do it?

Reply:
Yes, it's called prettier.io. I like it quite a bit. I make sure to always pass `--prose-wrap always` at the CLI so that it reflows paragraphs nicely.